plugins/index.js

```diff
@@ -11,6 +11,7 @@ let LabelsPlugin = require("./labels");
 let LandmarksPlugin = require("./landmarks");
 let LinkTextPlugin = require("./link-text");
 let A11yTextWand = require("./a11y-text-wand");
+let FocusTracker = require("./focus-tracker");
```

Comment:
I know you're just following the old style, but do you mind changing these `let`s to `const`?